src/Controller/UI/SearchPageController.php

```diff
@@ -23,14 +23,31 @@ class SearchPageController extends AbstractController
      */
     protected $logActionItemEventDispatcher;
 
+    /**
+     * Subsite name, based on custom template.
+     *
+     * @var string
+     */
+    protected $subSite;
+
     /**
      * Constructor for this Controller, to set up default services.
      *
      * @param LogActionItemEventDispatcher $logActionItemEventDispatcher The log action item event dispatcher.
      */
-    public function __construct(LogActionItemEventDispatcher $logActionItemEventDispatcher)
+    public function __construct(LogActionItemEventDispatcher $logActionItemEventDispatcher, ?string $customTemplate)
```

Comment:
Injectable should be never NULL, or else the framework throws an error, so the ? can go.